src/index.js

```diff
@@ -8,7 +8,7 @@ const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
   <React.StrictMode>
     <App />
-  </React.StrictMode>
+  </React.StrictMode>,
 );
```

Comment:
- [ ] Kindly make sure your application is running in the browser without errors, Actually, I'm getting this error message 👇🏻 Fix this

- Console:

![27compil](https://user-images.githubusercontent.com/79058364/176220146-27e28bc3-8d62-4202-ba9b-4f1d23b6404a.png)

- Browser:

![27compi](https://user-images.githubusercontent.com/79058364/176220250-e527549a-e8ba-40ff-a81a-c5b7ffe6ddcf.png)